mypy/stats.py

```diff
@@ -1,10 +1,9 @@
 """Utilities for calculating and reporting statistics about types."""
 
 import os.path
-import typing
 
 from collections import Counter
-from typing import Dict, List, cast, Optional
+from typing import cast, Counter, Dict, List, Optional
```

Comment:
This doesn't seem right. It overrides the import of `Counter` on the previous line. I think there's a reason why the code did what it did.